docs/en/latest/FAQ.md

```diff
@@ -569,3 +569,5 @@ The difference between `plugin-metadata` and `plugin-configs`:
 
 - Plugin instance scope: `plugin-metadata` works on all instances of this plugin. `plugin-configs` works on the plugin instances configured under it.
 - Binding entities: `plugin-metadata` take effect on the entities bound to all instances of this plugin. `plugin-configs` take effect on the routes bound to this `plugin-configs`.
+
+Note: plugin instances are plugins that are currently enabled
```

Comment:
I'm afraid the term `plugin instance` is not correct in this context, as the instance gives an illusion that there is more than one copy in Apache APISIX, but indeed we just have one object for each plugin.

Reply:
> but indeed we just have one object for each plugin.

Here I do not understand. For example, the `limit-req` plugin on different routes, is it the same object? Also you are saying that each plugin has only one object, only the `conf` is different.

Reply:
what about `plugin instance` -> `plugin conf`?

Reply:
update

Reply:
OK.